examples/README.md

```diff
@@ -0,0 +1,27 @@
+# Examples
+
+This folder contains finetuning and inference examples for Llama 2.
+
+## Finetuning
+
+Please refer to the main [README.md](../README.md) for information on how to use the [finetuning.py](./finetuning.py) script.
+After installing the llama-recipes package through [pip](../README.md#installation) you can also invoke the finetuning in two ways:
+```
+python -m llama_recipes.finetuning <parameters>
+
+python examnples/finetuning.py <parameters>
+```
+Please see [README.md](../README.md) for details.
+
+## Inference 
```

Comment:
Will be good to provide complete list and also mention the chat and codellama models here